public/src/client/topic/events.js

```diff
@@ -222,6 +225,10 @@ define('forum/topic/events', [
 		el.find('[component="post/bookmark/off"]').toggleClass('hidden', data.isBookmarked);
 	}
 
+	function toggleAnswered() {
+		console.log('hi');
```

Comment:
NIT: do you think we should change this to console log a more meaningful message?

Reply:
Thanks for catching that!